Dockerfile.worker

```diff
@@ -0,0 +1,40 @@
+# Build the manager binary
+FROM golang:1.20-alpine3.18 as builder
+
+RUN ["apk", "add", "bash", "make"]
+
+WORKDIR /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN ["go", "mod", "download"]
+
+# Copy the go source
+COPY api api
+COPY cmd/worker cmd/worker
+COPY internal internal
+
+# Copy Makefile
+COPY Makefile Makefile
+COPY docs.mk docs.mk
```

Comment:
Why do we need to docs in the binary?

Reply:
`Makefile` includes `docs.mk`, if that file is not present then `make` fails.